contracts/0.4.24/Lido.sol

```diff
@@ -285,8 +293,8 @@ contract Lido is Versioned, StETHPermit, AragonApp {
      * For more details see https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-10.md
      */
     function finalizeUpgrade_v2(address _lidoLocator, address _eip712StETH) external {
```

Comment:
want to preserve the same invariant as for the fresh deployment (non-zero balance for `0xdead`)